pkgs/by-name/ur/urh/package.nix

```
@@ -25,12 +30,36 @@ python3Packages.buildPythonApplication rec {
     pyqt5 numpy psutil cython pyzmq pyaudio setuptools
   ];
 
+  # dont double wrap
+  # https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-common-issues-double-wrapped
+  dontWrapGApps = true;
+  preFixup = ''
+    qtWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
   postFixup = ''
     wrapQtApp $out/bin/urh
   '';
 
   doCheck = false;
 
+  desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      exec = "urh";
+      icon = "urh";
+      desktopName = "Universal Radio Hacker";
+      categories = [ "HamRadio" ];
```

```suggestion
      categories = [ "Network" "HamRadio" ];
```
As an additional category, HamRadio should also have a main category. I would suggest Network, see https://specifications.freedesktop.org/menu-spec/latest/additional-category-registry.html.

---

Thanks, appreciated!